inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/core/impl/AuditServiceImpl.java

```diff
@@ -159,6 +167,55 @@ public Boolean refreshBaseItemCache() {
         return true;
     }
 
+    private AuditQuerySourceConfigEntity createAuditQuerySource(String sourceType, String sourceUrl,
+            Integer auth_Enable, String userName,
+            String password, String modifier) {
+        AuditQuerySourceConfigEntity auditQuerySourceConfig = new AuditQuerySourceConfigEntity();
+        auditQuerySourceConfig.setSourceType(sourceType);
+        auditQuerySourceConfig.setSourceUrl(sourceUrl);
+        auditQuerySourceConfig.setAuthEnable(auth_Enable);
+        auditQuerySourceConfig.setUserName(userName);
+        auditQuerySourceConfig.setPassword(password);
+        auditQuerySourceConfig.setStatus(1);
+        auditQuerySourceConfig.setCreator(modifier);
+        auditQuerySourceConfig.setCreateTime(new Date());
+        auditQuerySourceConfig.setModifier(modifier);
+        auditQuerySourceConfig.setModifyTime(new Date());
+        auditQuerySourceConfig.setIsDeleted(0);
+        auditQuerySourceConfig.setVersion(1);
+        return auditQuerySourceConfig;
+    }
+    @Override
+    public Boolean updateAuditQuerySource(AuditSourceRequest request) {
+        String oldUrl = request.getOldUrl();
+        String sourceType = request.getSourceType();
+        String sourceUrl = request.getSourceUrl();
+        String userName = request.getUserName();
+        String password = request.getPassword();
+        Integer authEnable = (request.getAuthEnable() == null) ? 1 : request.getAuthEnable();
+        String modifier = request.getModifier();
+        try {
+            if (!StringUtils.isBlank(oldUrl)) {
+                querySourceConfigEntityMapper.offlineAuditQuerySourceByUrl(oldUrl);
+            }
+            AuditQuerySourceConfigEntity entity =
+                    createAuditQuerySource(sourceType, sourceUrl, authEnable, userName, password, modifier);
+            querySourceConfigEntityMapper.insert(entity);
+            config.updateCkSource();
+        } catch (Exception e) {
+            LOGGER.error("fail to update audit query source!");
+            LOGGER.error(e.toString());
```

Comment:
Combine them into one log, and better to print the stack of exception
```suggestion
           LOGGER.error("fail to update audit query source.", e);
```